_commands/basics/kill.md

```diff
@@ -17,8 +18,32 @@ TODO: Add documentation for this command by submitting a pull request.
 
 #### Example command
 
-##### Break it down
+##### kill -9 PID
+Actually, kill is a command to send signals to the specified process, each process has a set of standard handlers by the operating systems, and signal 9 called SIGKILL can almost guarantee to kill a process if the default signal 15 can't kill it.
 
 #### Example command
 
-##### Break it down
+##### kill -l 
+Although kill -9 PID is the most common kill commands and you probably don't need to know more, but if you really need to know the full list of about 60 different signals, type kill -l
+
+#### Example command
+
+##### ps -aux | less 
```

Comment:
could you show example output of this command, and what value would correspond to the process ID?